monai/transforms/utility/array.py

```diff
@@ -128,6 +129,18 @@ def __call__(self, img: NdarrayOrTensor) -> NdarrayOrTensor:
         return img
 
 
+class RandIdentity(RandomizableTransform):
```

Comment:
`RandomizableTrait` is the minimal class...
```suggestion
class RandIdentity(RandomizableTrait):
```